src/adapters/openai/completions.ts

```diff
@@ -42,11 +43,21 @@ When making changes to plugin configurations, maintain this structure:
 
 # Example of correct plugin formatting
 - uses:
-- plugin: ubiquibot/issue-comment-embeddings@main
+- plugin: <ORG/OWNER>/<REPO>@main
   with:
     property1: value1
     property2: value2
 
+PLUGIN INSTRUCTIONS:
+- Ensure all plugin configurations are correctly formatted
+- Use the manifests below to understand valid plugin properties and default values
+- Do not remove any existing plugin configurations unless instructed
+- Add new plugin configurations at the end of the file
+- Infer ORG/OWNER and REPO details from the included plugin configurations and manifests
+- DO NOT REMOVE CONTENT UNLESS SPECIFICALLY INSTRUCTED TO DO SO
+- ALWAYS TARGET MAIN BRANCH FOR PLUGIN CONFIGURATIONS, UNLESS SPECIFIED OTHERWISE.
```

Comment:
This is not always true. Default branch is always true. I'm pretty sure our default branch on @ubiquity is `development`.

Reply:
Removed this. I was under the assumption that @ubiquity uses `main`.